cocos/dragon-bones/ArmatureDisplay.ts

```diff
@@ -57,7 +57,7 @@ ccenum(DefaultAnimsEnum);
 enum DefaultCacheMode {
     REALTIME = 0,
 }
-ccenum(DefaultAnimsEnum);
+ccenum(DefaultCacheMode);
```

Comment:
Why modify it?

Reply:
It's a bug. ccenum here is decorating `DefaultCacheMode` rather than `DefaultAnimsEnum`.
Normally, in our engine code, `ccenum` invocation is after enum definition.